### PR TITLE
Added install target, yeah! Cmake rules for Parser regeneration

### PR DESCRIPTION
**Pull request by alainmarcel**

### CODE REVIEW

File: src/SourceCompile/SV3_1aTreeShapeListener.h [hunk 1]
```diff
@@ -30,11 +30,11 @@
 #include <stack>
 #include <map>
 #include <unordered_map>
-#include "Utils/ParseUtils.h"
-#include "SourceCompile/SymbolTable.h"
-#include "Design/TimeInfo.h"
-#include "Design/DesignElement.h"
-#include "Design/FileContent.h"
+#include "../Utils/ParseUtils.h"
```
Comment (hzeller) [inline src/SourceCompile/SV3_1aTreeShapeListener.h]: Don't add ../, I just fixed that :)

Comment (hzeller) [reply src/SourceCompile/SV3_1aTreeShapeListener.h]: You probably need a `git pull --rebase` to get the latest from upstream.

Comment (alainmarcel) [reply src/SourceCompile/SV3_1aTreeShapeListener.h]: Those files are auto generated, let me update the script that generates them

**Comment by alainmarcel** [comment]
Fixes #66

**Pull request merged**